llama_stack/apis/common/content_types.py

```diff
@@ -30,21 +30,27 @@ def validator(cls, values):
             return values
         return {"url": values}
 
-    @field_serializer("data")
-    def serialize_data(self, data: Optional[bytes], _info):
-        if data is None:
-            return None
-        return base64.b64encode(data).decode("utf-8")
-
 
 @json_schema_type
 class ImageContentItem(BaseModel):
+    """A image content item
+
+    :param type: Discriminator type of the content item. Always "image"
+    :param image: Image as a base64 encoded string or an URL
```

Comment:
you can also point out that URLs could be data URLs but they have length limits.

Reply:
done